examples/storage/createjob.go

```diff
@@ -15,26 +14,20 @@ import (
 )
 
 func main() {
-	keyID := os.Getenv("SDC_KEY_ID")
-	accountName := os.Getenv("SDC_ACCOUNT")
-	keyPath := os.Getenv("SDC_KEY_FILE")
+	keyID := os.Getenv("MANTA_KEY_ID")
+	accountName := "tritongo"
```

Comment:
Let's use `os.Getenv("MANTA_USER")` as `accountName` here so it works out of the box.